styles/index.css

```diff
@@ -20,6 +20,16 @@ html {
 body {
   @apply bg-gray-100;
 }
+
+#__next {
```

Comment:
The combination of #__next with these styles and `Main` with `flex-grow` gives us the "pancake layout":